core_backend/tests/api/conftest.py

```diff
@@ -41,11 +41,14 @@ def faq_contents(client: TestClient) -> None:
     points = []
     for content in json_data:
         point_id = str(uuid.uuid4())
+        text_to_embed = content["content_title"] + "\n" + content["content_text"]
```

Comment:
For a future PR: Wonder if there is anyway to use the titles more effectively. E.g. Do sparse search on titles and dense on content?

Reply:
Added backlog ticket :)